CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,9 @@
+# Microsoft Open Source Code of Conduct
```

Comment:
Is this file necessary? Should we either add it to all the vscode-remote-try-* repos, or remove it from this one, for consistency?

Reply:
It's part of the new oss repository templates. And according to my experience, this has proven to be very useful is several occasions (I can explain them in Teams). 

I would recommend to add them on all repos. It's something my team can help with.

Reply:
Thank you! I think we'd want to add those all at the same time so that it doesn't seem we're treating different repos differently. If you'd be able to open PRs to add them to the repos in https://github.com/search?q=org%3Amicrosoft+vscode-remote-try-&type=Repositories, that'd be a huge help.

Reply:
Done 🚀